content/patterns/button/button-pattern.html

```diff
@@ -14,92 +14,93 @@
   </head>
   <body>
     <main>
-      <h1>Button Pattern</h1>
+      <h1>ボタン パターン</h1>
+      <!-- 翻訳元リビジョン: https://github.com/w3c/aria-practices/tree/d6cf9d8db772603a456e7d7ef52e409d39124301 -->
 
       <section id="about">
-        <h2>About This Pattern</h2>
+        <h2>このパターンについて</h2>
         <p>
-          A <a class="role-reference" href="#button">button</a> is a widget that enables users to trigger an action or event, such as submitting a form, opening a dialog, canceling an action, or performing a delete operation.
-          A common convention for informing users that a button launches a dialog is to append &quot;&#8230;&quot; (ellipsis) to the button label, e.g., &quot;Save as&#8230;&quot;.
+          <a class="role-reference" href="#button">ボタン</a> は、フォームの送信、ダイアログのオープン、アクションのキャンセル、削除操作などの利用者のアクションやイベントのトリガを可能にするウィジェットです。
+          ボタンがダイアログを起動することを利用者に知らせるための一般的な方法は、ボタンラベルに &quot;&#8230;&quot;（省略記号）を付加することです。例： &quot;名前を付けて保存&#8230;&quot;
         </p>
-        <p>In addition to the ordinary button widget, WAI-ARIA supports 2 other types of buttons:</p>
+        <p>通常のボタンウィジェットに加えて、WAI-ARIA は2種類のボタンをサポートしています：</p>
         <ul>
           <li>
-            Toggle button: A two-state button that can be either off (not pressed) or on (pressed).
-            To tell assistive technologies that a button is a toggle button, specify a value for the attribute <a href="#aria-pressed" class="state-reference">aria-pressed</a>.
-            For example, a button labelled mute in an audio player could indicate that sound is muted by setting the pressed state true.
-            <strong>Important:</strong> it is critical the label on a toggle does not change when its state changes.
-            In this example, when the pressed state is true, the label remains &quot;Mute&quot; so a screen reader would say something like &quot;Mute toggle button pressed&quot;.
-            Alternatively, if the design were to call for the button label to change from &quot;Mute&quot; to &quot;Unmute,&quot; the aria-pressed attribute would not be needed.
+            トグルボタン：オフ（押されていない状態）とオン（押された状態）の2つの状態を持つボタン。
+            ボタンがトグルボタンであることを支援技術に伝えるには、<a href="#aria-pressed" class="state-reference">aria-pressed</a> 属性に値を指定します。
+            例えば、オーディオプレーヤーのミュートというラベルのボタンは、押された状態を true に設定することで、音がミュートされていることを示すことができます。
+            <strong>重要：</strong>トグルボタンのラベルは、その状態が変化しても変化しないことが重要です。
+            この例では、押された状態が true のとき、ラベルは &quot;ミュート&quot; のままなので、スクリーンリーダーは "ミュート トグルボタン 押されています" のように言うでしょう。
```

Comment:
スクリーンリーダーは「言う」より「読み上げる」の方が一般的な表現な気がしました。

```suggestion
            この例では、押された状態が true のとき、ラベルは &quot;ミュート&quot; のままなので、スクリーンリーダーは "ミュート トグルボタン 押されています" のように読み上げるでしょう。
```